UPGRADE-5.3.md

```diff
@@ -43,6 +43,7 @@ FrameworkBundle
  * Deprecate the `session` service and the `SessionInterface` alias, use the `\Symfony\Component\HttpFoundation\Request::getSession()` or the new `\Symfony\Component\HttpFoundation\RequestStack::getSession()` methods instead
  * Deprecate the `KernelTestCase::$container` property, use `KernelTestCase::getContainer()` instead
  * Rename the container parameter `profiler_listener.only_master_requests` to `profiler_listener.only_main_requests`
+ * Deprecate option `--xliff-version` of the `translation:update` command, use e.g. `--output-format=xlf20` instead
```

Comment:
Shouldn't it be `--output=xlf20` (same below)?

Reply:
Yes, indeed. Same in Translation CHANGELOG.md. I also moved "* Add `translation:pull` and `translation:push` commands to manage translations with third-party providers" from FrameworkBundle CHANGELOG to Translation CHANGELOG